src/index.js

```diff
@@ -1,60 +1,103 @@
+import {loadPeople, loadPlanets, loadFilms, loadSpecies, loadVehicles} from './loadData.js';
+
+let category;
+let categorySelected = false;
+let currentPageInfo;
+
 const cardsField = document.querySelector('.card-body');
 const categories = document.querySelectorAll(`.nav-link`);
 const paginator = document.createElement('ul');
-const loader = document.querySelector('.starwars-demo')
-let category;
+const introImage = document.querySelector('.starwars-demo');
+const loader = document.querySelector('.lds-facebook');
+const search = document.getElementById('search');
+const searchField = document.querySelector('.search-field');
+const save = document.getElementById('save');
+const load = document.getElementById('load');
+const themeSwitcher = document.getElementById('customSwitch1');
+
+
+window.onload = function loadSavedData() {
+    let theme = localStorage.getItem('theme');
+
+    if (theme === 'dark') {
+        themeSwitcher.checked = true;
+    } else {
+        themeSwitcher.checked = false;
+    }
+    changeTheme();
+
+}
 
 categories.forEach(el => {
     el.addEventListener('click', (e) => {
         categories.forEach(el => {
             if(el.classList[2]){
             el.classList.remove('selected') 
         }});
+
         e.target.classList.add('selected');
         category = el.innerHTML;
+
         loadCategory(el.innerHTML, 1);
+
     });
 })
 
+paginator.addEventListener('click', moveToPage);
+search.addEventListener('click', searchValue);
+save.addEventListener('click', saveData);
+load.addEventListener('click', loadData);
+themeSwitcher.addEventListener('click', changeTheme);
 
 function loadCategory(category, page) {
     cardsField.innerHTML = '';
-    loader.style.display = 'flex';
+    introImage.style.display = 'none';
+    loader.style.display = 'inline-block';
+    categorySelected = true;
+
     fetch(`https://www.swapi.tech/api/${category}?page=${page}&limit=10`)
         .then(res => res.json())
         .then(data => loadDetails(data, category, page))
         .then(() => loader.style.display = 'none')
         .catch(err => notifyByError(err.status));
```

Comment:
You can write just `.catch(notifyByError)`

Reply:
Thank you for advice, made refactoring here.